grakn-graql/src/main/java/ai/grakn/graql/internal/reasoner/query/ReasonerAtomicQuery.java

```diff
@@ -296,19 +306,31 @@ public Unifier getUnifier(ReasonerQuery p){
      * @param materialise whether inferred information should be materialised
      * @return stream of differential answers
      */
-    public Stream<Answer> answerStream(Set<ReasonerAtomicQuery> subGoals,
+    Stream<Answer> answerStream(Set<ReasonerAtomicQuery> subGoals,
                                        Cache<ReasonerAtomicQuery, ?> cache,
                                        Cache<ReasonerAtomicQuery, ?> dCache,
                                        boolean materialise,
                                        boolean explanation,
                                        boolean differentialJoin){
         boolean queryAdmissible = !subGoals.contains(this);
 
+        LOG.trace("AQ: " + this);
```

Comment:
Atomic Query :) This is intentionally a shorthand to make the logs more readable.